src/fixup-svg-string.js

```diff
@@ -57,5 +57,10 @@ module.exports = function (svgString) {
     // Empty script tags and javascript executing
     svgString = svgString.replace(/<script[\s\S]*>[\s\S]*<\/script>/, '<script></script>');
 
+    // Empty script tags and javascript executing
+    svgString = svgString.replace(/<script[\s\S]*>[\s\S]*<\/script>/, '<script></script>');
```

Comment:
Is this intentional? (If we want proper escaping we should just move DOMPurify here)